src/Symfony/Component/Console/Application.php

```diff
@@ -769,8 +769,8 @@ protected function configureIO(InputInterface $input, OutputInterface $output)
 
         if (true === $input->hasParameterOption(array('--no-interaction', '-n'), true)) {
             $input->setInteractive(false);
-        } elseif (function_exists('posix_isatty') && $this->getHelperSet()->has('question')) {
-            $inputStream = $this->getHelperSet()->get('question')->getInputStream();
```

Comment:
This should probably not be removed entirely yet ?
Or should `QuestionHelper::setInputStream` && `QuestionHelper::getInputStream` methods call `$input->setStream` && `$input->getStream` ?